app/client/src/routes.js

```diff
@@ -94,6 +95,9 @@ angular.module('reg')
           }
         }
       })
+      .state('app.sponsorRequest', {
+        // Do some stuff here... email corporate team account info?
+      })
```

Comment:
nit: remove from this PR -- add "request to be sponsor" functionality in a different PR